extmod/nimble/modbluetooth_nimble.c

```diff
@@ -949,6 +949,16 @@ int mp_bluetooth_gap_peripheral_connect(uint8_t addr_type, const uint8_t *addr,
     return ble_hs_err_to_errno(err);
 }
 
+int mp_bluetooth_gap_peripheral_connect_cancel() {
```

Comment:
Should be `(void)`  (also in modbluetooth.h and modbluetooth_btstack.c)

Reply:
This is returning the errno from the stack, though, same as the other `mp_bluetooth_` methods.

Reply:
Sorry, I mean

```c
int mp_bluetooth_gap_peripheral_connect_cancel(void);
```

Otherwise C treats "void foo();" as a function that takes any number of arguments.

Reply:
Oops! Yes, of course, sorry! I'm slightly embarrassed to admit that my C stretches back to K&R and I can be a bit slapdash about the "modern" conventions ;-)